nodejs/nodeapi-polyfill.js

```diff
@@ -0,0 +1,86 @@
+var crypto = require('crypto');
+crypto.createECDH = require('create-ecdh');
+
+Buffer.prototype.equals = function equals(b) {
```

Comment:
Is there an npm package that covers this?  I'd rather not take on the job of maintaining this.